README.md

```diff
@@ -1,27 +1,41 @@
-# React + TypeScript + Vite
-
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
-
-Currently, two official plugins are available:
-
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+# basic_ecom
+
+A basic e-commerce React project used on my mentorship programme.
+
+## Technologies
+
+- [Git](https://git-scm.com/) - Version control system
+- [Node](https://nodejs.org/en) - Runtime
+- [NPM](https://www.npmjs.com/) - Package manager
+- [Husky](https://typicode.github.io/husky/) - Git hook helper
+- [commitlint](https://commitlint.js.org/) - Lint commit messages
+- [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) - Specification for commit messages
+- [lint-staged](https://github.com/okonet/lint-staged/) - Run linters and formatters against staged git files
+- [ESLint](https://eslint.org/) - Linter
+- [Prettier](https://prettier.io/) - Formatter
+- [Vite](https://vitejs.dev/) - Bundler
+- [React](https://react.dev/) - Framework
+- [Typescript](https://www.typescriptlang.org/) - Static types
+- [React Router](https://reactrouter.com/en/main) - Routing
+- [Tailwindcss](https://tailwindcss.com/) - Style
+
+## Installation
+
+- Clone repo and cd into project
+
+```sh
+$ git clone git@github.com:tamara24github/basic_ecom.git
+$ cd basic_ecom
+```
 
-## Expanding the ESLint configuration
+- Make sure to use the node version specified in `.nvmrc`. It is recommended to use `nvm` for node version management. [Official nvm docs](https://github.com/nvm-sh/nvm/blob/master/README.md)
```

Comment:
You are mentioning the .nvmrc file here but the file is not present in the pr, you should add it. This is described in the task.